site/docs/master/run-locally.md

```diff
@@ -0,0 +1,110 @@
+# Run Velero locally in development
+
+Running the Velero server locally can speed up iterative development. This eliminates the need to rebuild the Velero server
+image and redeploy it to the cluster with each change.
+
+## Run Velero locally with a remote cluster
+
+Velero runs against the Kubernetes API server as the endpoint (as per the `kubeconfig` configuration), so both the Velero server and client use the same `client-go` to communicate with Kubernetes. This means the Velero server can be run locally just as functionally as if it was running in the remote cluster.
+
+### Prerequisites
+
+When running Velero, you will need to ensure that you set up all of the following:
+
+* Appropriate RBAC permissions in the cluster
+  * Read access for all data from the source cluster and namespaces
+  * Write access to the target cluster and namespaces
+* Cloud provider credentials
+  * Read/write access to volumes
+  * Read/write access to object storage for backup data
+* A [BackupStorageLocation][20] object definition for the Velero server
+* (Optional) A [VolumeSnapshotLocation][21] object definition for the Velero server, to take PV snapshots
+
+### 1. Create a cluster
```

Comment:
This makes it sound like you need a cluster on AWS to run velero locally, perhaps we should just link to the Install section which already goes through various cluster options on different platforms (including local) instead of pointing out specific options here?